cloud-pubsub/main.py

```diff
@@ -6,13 +6,23 @@
 """
 
 import datetime
+import time
 
 from google.cloud import pubsub
 
 PUBSUB_TOPIC = 'eco'
 PUBSUB_SUBSCRIPTION = 'echo-read'
 
 
+def process(message):
```

Comment:
please consider moving below main() as that's the actual thing we want to show.

Reply:
done